docs/wtf-is-metagame/300-of-metagame.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: 💪 The 300 of MetaGame.md
```

Comment:
Nice work on these @amosgoh 🥳 Would you do us a favour and remove the '.md' from this title? 🙏🏻